code/modules/mob/living/carbon/human/human.dm

```diff
@@ -1626,9 +1630,6 @@
 	if(istype(head, /obj/item/clothing/head/wizard) || istype(head, /obj/item/clothing/head/helmet/space/rig/wizard))
 		threatcount += 2
 
-	//Check for nonhuman scum
-	if(dna && dna.mutantrace && dna.mutantrace != "none")

```

Comment:
Functionality loss, should probably change to checking for species != "human", as that was the intention of the code